dev/ci/ci-basic-overlay.sh

```diff
@@ -125,5 +125,5 @@
 ########################################################################
 # Bignums
 ########################################################################
-: ${bignums_CI_BRANCH:=master}
-: ${bignums_CI_GITURL:=https://github.com/coq/bignums.git}
+: ${bignums_CI_BRANCH:=fix-thunk-printer}
+: ${bignums_CI_GITURL:=https://github.com/ppedrot/bignums.git}
```

Comment:
What the heck!?!?!?!?!?!?!?!?
That's **not** how you use overlays.
Overlays for testing a PR **must** go to [`dev/ci/user-overlays`](dev/ci/user-overlays).
There have been a bad habit of changing this file [`ci-basic-overlay.sh`](/dev/ci/ci-basic-overlay.sh) to create lasting overlays but that's wrong and we should strive to remove them. (Because each lasting overlay corresponds to a development that we think builds with master but actually doesn't.)
In the case of bignums, we control the repo, so lasting overlays make absolutely no sense!
That being said, the kind of breaking change introduced by this PR should have implied that bignums would now contain a master branch following Coq master, and a v8.7 branch following Coq v8.7.